receiver/prometheusreceiver/internal/metricsbuilder.go

```diff
@@ -124,7 +124,8 @@ func (b *metricBuilder) AddDataPoint(ls labels.Labels, t int64, v float64) error
 	return b.currentMf.Add(metricName, ls, t, v)
 }
 
-// Build is to build an opencensus data.MetricsData based on all added data complexValue
+// Build is to build an opencensus data.MetricsData based on all added data complexValue.
+// The only error returned by this function is errNoDataToBuild.
```

Comment:
Later we should look at returning a bool instead of an error but didn't want to make a bigger change now.